client/src/components/molecules/AdminSettings/index.tsx

```diff
@@ -25,27 +43,32 @@ const AdminSettings = ({ isOpen, setIsOpen }: { isOpen: boolean, setIsOpen: () =
       >
         {menu}
       </button>
-    )
+    );
   });
 
-  const [passwordData, setPasswordData] = useState<{
-    currentPassword: string
-    newPassword: string
-    newConfirmedPassword: string
-  }>({
+  const [passwordData, setPasswordData] = useState<Password>({
     currentPassword: "",
     newPassword: "",
     newConfirmedPassword: ""
   });
```

Comment:
Can we try it like this?
```suggestion
  const [passwordData, setPasswordData] = useState<Password | null>(null);
```